subgraphs/abracadabra/src/common/setters.ts

```diff
@@ -0,0 +1,88 @@
+      MarketEntity.maximumLTV = bigIntToBigDecimal(BigInt.fromI32(75000), 5);
+      MarketEntity.liquidationPenalty = bigIntToBigDecimal(BigInt.fromI32(112500), 5).minus(BIGDECIMAL_ONE);
+      MarketEntity.liquidationThreshold = bigIntToBigDecimal(BigInt.fromI32(75000), 5); // ???
+      MarketEntity.stableBorrowRate = bigIntToBigDecimal(BigInt.fromI32(475331078), 18).times(SECONDS_PER_YEAR);
+    } else if (marketAddress.toLowerCase() == "0xffbf4892822e0d552cff317f65e1ee7b5d3d9ae6".toLowerCase()) {
+      MarketEntity.maximumLTV = bigIntToBigDecimal(BigInt.fromI32(75000), 5);
+      MarketEntity.liquidationPenalty = bigIntToBigDecimal(BigInt.fromI32(112500), 5).minus(BIGDECIMAL_ONE);
+      MarketEntity.liquidationThreshold = bigIntToBigDecimal(BigInt.fromI32(75000), 5); // ???
+      MarketEntity.stableBorrowRate = bigIntToBigDecimal(BigInt.fromI32(475331078), 18).times(SECONDS_PER_YEAR);
+    } else if (marketAddress.toLowerCase() == "0x6cbafee1fab76ca5b5e144c43b3b50d42b7c8c8f".toLowerCase()) {
+      MarketEntity.maximumLTV = bigIntToBigDecimal(BigInt.fromI32(100000), 5);
+      MarketEntity.liquidationPenalty = bigIntToBigDecimal(BigInt.fromI32(103000), 5).minus(BIGDECIMAL_ONE);
+      MarketEntity.liquidationThreshold = bigIntToBigDecimal(BigInt.fromI32(75000), 5); // ???
+      MarketEntity.stableBorrowRate = bigIntToBigDecimal(BigInt.fromI32(253509908), 18).times(SECONDS_PER_YEAR);
+    } else if (marketAddress.toLowerCase() == "0xbb02a884621fb8f5bfd263a67f58b65df5b090f3".toLowerCase()) {
+      MarketEntity.maximumLTV = bigIntToBigDecimal(BigInt.fromI32(75000), 5);
+      MarketEntity.liquidationPenalty = bigIntToBigDecimal(BigInt.fromI32(112500), 5).minus(BIGDECIMAL_ONE);
+      MarketEntity.liquidationThreshold = bigIntToBigDecimal(BigInt.fromI32(75000), 5); // ???
+      MarketEntity.stableBorrowRate = bigIntToBigDecimal(BigInt.fromI32(475331078), 18).times(SECONDS_PER_YEAR);
+    } else {
+      let maximumLTVCall = MarketContract.try_COLLATERIZATION_RATE();
+      let liquidationPenaltyCall = MarketContract.try_LIQUIDATION_MULTIPLIER();
+      let accrueInfoCall = MarketContract.try_accrueInfo();
+      if (!maximumLTVCall.reverted && !liquidationPenaltyCall.reverted && !accrueInfoCall.reverted) {
+        MarketEntity.maximumLTV = bigIntToBigDecimal(maximumLTVCall.value, 5);
+        MarketEntity.liquidationPenalty = bigIntToBigDecimal(liquidationPenaltyCall.value, 5).minus(BIGDECIMAL_ONE);
+        MarketEntity.liquidationThreshold = bigIntToBigDecimal(maximumLTVCall.value, 5); // ???
+        MarketEntity.stableBorrowRate = bigIntToBigDecimal(accrueInfoCall.value.value2, 18).times(SECONDS_PER_YEAR);
+      }
+    }
+    MarketEntity.depositRate = BIGDECIMAL_ZERO;
+    MarketEntity.variableBorrowRate = BIGDECIMAL_ZERO; // ???
+  }
+  MarketEntity.save();
+  updateProtocolMarketList(marketAddress);
+}
+
+export function createCachedLiquidation(event: LogRemoveCollateral): void {
+  let liquidation = new _LiquidationCache(
```

Comment:
Would you be able to build entities IDs like this https://github.com/messari/subgraphs/commit/0f8a5def6f3331128ce832b2b5c858dd2f492e23